pvnet_app/data/satellite.py

```diff
@@ -343,30 +346,30 @@ def preprocess_sat_data(t0: pd.Timestamp, use_legacy: bool = False) -> pd.Dateti
     extend_satellite_data_with_nans(t0)
 
     # Check for zeros in the satellite data
-    check_for_zeros()
+    check_for_constant_values()
```

Comment:
Will this still pass at night time? I'm not sure if the visible channels go exactly to zero or not

Reply:
this seems ok. They dont seem to quite get to zero, but they do get low